### PR TITLE
article viewing feature cucumber

### PR DESCRIPTION
**Pull request by irchik**
need review

### CODE REVIEW

File: features/functionality/articles/viewing.feature [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
       | text      | UNIQ_ARTICLE[:text]   |
+    And I am logged in as admin user
     And comment to UNIQ_ARTICLE[:title] article with parameter:
       | body      | UNIQ_COMMENT[:text]   |
 
-  Scenario: user is viewing article page
-    When I open UNIQ_ARTICLE[:title] article page     
-    Then I should see article parameters with data:
+  Scenario: admin is viewing article page
+    When I open article page for UNIQ_ARTICLE[:title]
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: When I navigate to UNIQ_ARTICLE[:title] article on article list page

File: features/functionality/articles/viewing.feature [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
+    And I should see admin user comment on article page with data:
+      | commenter | admin     |
       | comment   | UNIQ_COMMENT[:text]   |
     And I should see add comment form on article page
     And I should see body field on article page
     And I should see buttons: edit article, destroy comment, create comment on article page
-    When I click back to articles link on article page
-    Then I should be redirected to articles list page
\ No newline at end of file
+    When I click Back to Articles link on article page
```
Comment (romikoops) [inline features/functionality/articles/viewing.feature]: Why you changed it from lower letter to capital letter? Please revert and fix definition

File: features/functionality/articles/viewing.feature (hunk 1 above)
Comment (irchik) [inline features/functionality/articles/viewing.feature]: fixed

File: features/functionality/articles/viewing.feature (hunk 2 above)
Comment (irchik) [inline features/functionality/articles/viewing.feature]: fixed

File: pages/article_page.rb [hunk 3]
```diff
@@ -34,4 +36,43 @@ def click_article_button(text)
...
+  end
+
+  def destroy_comment_link_present?
+    find("[href*='comment']")
+    true
+  rescue Capybara::ElementNotFound
+    false
+  end
+
+  def navigate_to_page()
```
Comment (romikoops) [inline pages/article_page.rb]: what page? article_list?

def back_to_article_list

File: pages/login_page.rb [hunk 4]
```diff
@@ -36,7 +36,7 @@ def login_as(email, password, remember_me=false)
     HomePage.given
   end
 
-  def navigate_to_forgot_password_page
+  def navigate_to_page
```
Comment (romikoops) [inline pages/login_page.rb]: Why did you renamed it?

File: spec/bvt/functionality/articles/viewing_spec.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    expect(ArticlePage.given.text).to include(@comment.text)
+    expect(ArticlePage.given).to be_comment_form_present
+    expect(ArticlePage.given).to be_body_field_present
+    expect(ArticlePage.given).to be_edit_button_present
+    expect(ArticlePage.given).to be_add_comment_button_present
+    expect(ArticlePage.given).to be_destroy_comment_link_present
+  end
+
+  scenario "Admin can be redirected from article page back to article list" do
+    ArticlePage.given.navigate_to_page
```
Comment (romikoops) [inline spec/bvt/functionality/articles/viewing_spec.rb]: +1

File: spec/bvt/functionality/auth/password_resetting_spec.rb [hunk 6]
```diff
@@ -36,7 +36,7 @@
 
   scenario "user can not reset password with incorrect email" do
     LoginPage.open.
-        navigate_to_forgot_password_page
+        navigate_to_page
```
Comment (romikoops) [inline spec/bvt/functionality/auth/password_resetting_spec.rb]: +1

File: spec/bvt/functionality/auth/password_resetting_spec.rb [hunk 7]
```diff
@@ -47,7 +47,7 @@
     user = Gen.user
     sign_up_as(user)
     LoginPage.open.
-        navigate_to_forgot_password_page
+        navigate_to_page
```
Comment (romikoops) [inline spec/bvt/functionality/auth/password_resetting_spec.rb]: +1

File: spec/support/password_resetting_helper.rb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 module PasswordResettingHelper
   def user_restores_password(email)
     LoginPage.open.
-        navigate_to_forgot_password_page
+        navigate_to_page
```
Comment (romikoops) [inline spec/support/password_resetting_helper.rb]: +1

File: pages/article_page.rb (hunk 3 above)
Comment (irchik) [inline pages/article_page.rb]: this function used also for forgot|_password_page and article_list page, so i decided to use common name  for function. if you think that it is better to separate this fuintions - i`ll do it

File: pages/login_page.rb (hunk 4 above)
Comment (irchik) [inline pages/login_page.rb]: this funtion used in the same step with article

**Pull request merged**